### PR TITLE
magento/devdocs#: Replace <your Magento install dir> with <magento_root>

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

Per @dobooth , notice in https://github.com/magento/devdocs/pull/4274 we should use `<magento_root>` instead of `<your Magento install dir>` in the documentation.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.0/install-gde/install/sample-data-after-composer.html
- https://devdocs.magento.com/guides/v2.1/install-gde/install/sample-data-after-composer.html
- https://devdocs.magento.com/guides/v2.2/install-gde/install/sample-data-after-composer.html
- https://devdocs.magento.com/guides/v2.3/install-gde/install/sample-data-after-composer.html
- https://devdocs.magento.com/guides/v2.0/config-guide/config/config-files.html
- https://devdocs.magento.com/guides/v2.1/config-guide/config/config-files.html
- https://devdocs.magento.com/guides/v2.2/config-guide/config/config-files.html
- https://devdocs.magento.com/guides/v2.3/config-guide/config/config-files.html
- https://devdocs.magento.com/guides/v2.0/comp-mgr/trouble/cman/cron.html
- https://devdocs.magento.com/guides/v2.1/comp-mgr/trouble/cman/cron.html
- https://devdocs.magento.com/guides/v2.2/comp-mgr/trouble/cman/cron.html
- https://devdocs.magento.com/guides/v2.3/comp-mgr/trouble/cman/cron.html
- https://devdocs.magento.com/guides/v2.0/install-gde/trouble/readiness/tshoot_rc_cron.html
- https://devdocs.magento.com/guides/v2.1/install-gde/trouble/readiness/tshoot_rc_cron.html
- https://devdocs.magento.com/guides/v2.2/install-gde/trouble/readiness/tshoot_rc_cron.html
- https://devdocs.magento.com/guides/v2.3/install-gde/trouble/readiness/tshoot_rc_cron.html
- https://devdocs.magento.com/guides/v2.0/comp-mgr/trouble/cman/component-depend.html
- https://devdocs.magento.com/guides/v2.1/comp-mgr/trouble/cman/component-depend.html
- https://devdocs.magento.com/guides/v2.2/comp-mgr/trouble/cman/component-depend.html
- https://devdocs.magento.com/guides/v2.3/comp-mgr/trouble/cman/component-depend.html

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: _includes/install/file-system-perms-twouser_22.md [hunk 1]
```diff
@@ -105,12 +105,26 @@ To set ownership and permissions before you install the Magento software:
...
-		find var generated vendor pub/static pub/media app/etc -type f -exec chmod g+w {} +
-		find var generated vendor pub/static pub/media app/etc -type d -exec chmod g+ws {} +
-		sudo chown -R :<web server group> .
-		chmod u+x bin/magento
+    
+    ```bash
+	cd <magento_root>
+	```
+	
+	```bash
```
Comment (rogyar) [inline _includes/install/file-system-perms-twouser_22.md]: I have doubts about wrapping every command into a separate tag. However, they should be executed step by step. So, it's ok.

Comment (atwixfirster) [reply _includes/install/file-system-perms-twouser_22.md]: Thank you for reviewing, @rogyar .

Regarding @dobooth [comment](https://github.com/magento/devdocs/pull/4274#issuecomment-484224801) such format is ok. 

Thank you!

**Comment by dobooth** [comment]
running tests

**Pull request merged**